staff/sergio-luz/mailchimp/index.html

```diff
@@ -0,0 +1,135 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body class="wrapper">
+
+    <header class="wrapper header">
+        <div class="information-panel">
+            <div class="information-panel__left">
+                <img class="chimp" src="/staff/sergio-luz/mailchimp/images/easy-email-newsletters-banner-png8.png" alt="">
+            </div>
+            <div class="information-panel__right">
+                <h1 class="title">Easy Email <br> Newsletters</h1>
+                <p class="information-panel__right__text">MailChimp helps you design email newsletters, share them on
+                    social networks, integrate with services you already use, and track your results. It's like
+                    your personal publishing platform.</p>
+                <button for="SingUp" class="button--main-orange button">Sign Up Free</button>
+
+            </div>
+        </div>
+        <div class="header__bar"> 
+            <div class="header__bar__top">
+                <p class="header__bar__top__login p--white">Log In</p>
+            </div>
+            <div class="header__bar__container">
+                <img class="header__bar__container--item-center logo" src="./images/main_logo.png" alt="">
+                <button for="SingUp" class="header__bar__container--item-center button">Sign Up Free</button>
+                <div class="header__bar__container--item-center nav">
+                    <p class="p--first-in-nav">Pricing</p>
+                    <hr class="br--vertical">
+                    <p>Features</p>
+                    <hr class="br--vertical">
+                    <p>Resources</p>
+                    <hr class="br--vertical">
+                    <p>Support</p>
+                    <hr class="br--vertical">
+                    <p>Blog</p>
+                    <input class="search--input" type="search" placeholder="   Search">
+                </div>
+            </div>
+        </div>
+
+    </header>
+
+    <div class="div-BGcenter"></div>
+    <div class="div-BGfooter"></div>
+    <main class="box box__main">
+        <div class="box__main__title">
+            <h1 class="box__main--text h1">What's new</h2>
+        </div>
+        <div class="box__main__container box__main--text">
+            <div class="main__items__container">
+                <img class="main__items__container__item" src="./images/home/templates.jpg" alt="">
+                <h4 class="h4">New feature:</h4>
+                <h3 class="h3">More Templates!</h3>
+                <p class="p">We just added a boatload of beautiful templates to MailChimp,and they're free for
+                    everyone.</p>
+            </div>
+            <div class="main__items__container">
```

Comment:
You shouldn't have so many nested elements like this. You should keep it as block__element. In this case you can probably treat these divs as new blocks. This comment applies elsewhere in the HTML file